container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -416,8 +416,9 @@ function generate_test_results {
     && (subunit2junitxml ${TEMPEST_LOGS_DIR}testrepository.subunit > ${TEMPEST_LOGS_DIR}tempest_results.xml || true) \
     && subunit2html ${TEMPEST_LOGS_DIR}testrepository.subunit ${TEMPEST_LOGS_DIR}stestr_results.html || true
 
-    # NOTE: Remove cirros image before copying of the logs.
+    # NOTE: Remove images before copying of the logs.
     rm ${TEMPEST_DIR}/etc/*.img
+    rm ${TEMPEST_DIR}/etc/*.qcow2
```

Comment:
A nit, otherwise looks good to me:)
```suggestion
    # NOTE: Remove images before copying of the logs.
    rm ${TEMPEST_DIR}/etc/*.{img,qcow2}
```

Reply:
done, thanks